apps/content-watcher/src/controllers/v1/search.controller.ts

```diff
@@ -12,7 +12,10 @@ export class SearchControllerV1 {
 
   @Post()
   @HttpCode(HttpStatus.OK)
-  @ApiOperation({ summary: 'Search for DSNP content by id, start/end block, and filters' })
+  @ApiOperation({
+    summary:
+      'Search for DSNP content by id and filters, starting from `upperBound` block and going back for `blockCount` number of blocks',
```

Comment:
Just a note; `upperBoundBlock` is not required; will default to latest block (as of when the search job actually begins to process) if not provided.